cogs/help.py

```diff
@@ -200,6 +223,46 @@ async def fake_send(_self, *args, **kwargs):
         FakeContext.send = fake_send
         await self.close(FakeContext(thread, thread_author, thread.guild))
 
+    
+    @tasks.loop(minutes=5.0)
```

Comment:
This should probably be upped quite a bit?

Reply:
it does that 15m check remember